grails-app/controllers/au/org/emii/portal/WpsController.groovy

```diff
@@ -63,7 +64,7 @@ class WpsController extends HostVerifyingController {
                 job: [
                     uuid: params.uuid,
                     reportUrl: g.createLink(action: 'jobReport', absolute: true, params: params),
-                    createdTimestamp: new DateTime(String.valueOf(execResponse.Status.@creationTime)),
+                    createdTimestamp: new DateTime(String.valueOf(execResponse.Status.@creationTime)).withZone( DateTimeZone.forID( "Australia/Melbourne" )),
```

Comment:
this commit, shoudln't be included here, if it's already in the other PR https://github.com/aodn/aodn-portal/pull/2356/commits/3501a7a8c05069ec7096bb0268ae353a27d27e7c